### PR TITLE
Metflix version 1.0.0

### PR DESCRIPTION
**Pull request by Zilola-Nazarova**
### This project follows instructions of [JavaScript capstone project - Your API-based webapp](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md) lesson.

**In the `dev` branch, we have the finalized version of the Metflix project ready for deployment.**

### General requirements

- there are no linter errors;
- correct Gitflow is used (`feature` -> `dev` -> `main`);
- the work is documented in a professional way regarding:
  - branches naming;
  - PR description;
  - commit messages;
  - node_modules (in `.gitignore`);
  - README.md file;
- we tried to follow the list of best practices for JavaScript, HTML and CSS as much as possible.

### Project requirements:

We created a new app using the external Involvement API and saved the returned ID of the app.
Currently, the ID we are using is: `IXNHPoRODtuVf9PEinhq`

**Required features:**

- "Search" button performs a search using external TVmaze API and retrieves the data about matching items (movies);
- `<main>` section of the page is created dynamically using the data retrieved from the TVmaze API;
- each item in the `<main>` contains a Like button (a heart icon):
  - a user can like any item in the `<main>`. This interaction is enabled using external Involvement API;
  - the number of likes is preserved using Involvement API and updated when a new like is given;
- "Comment" button under each item opens the pop-up window which displays:
  - the details of the selected item (using data from TVmaze API);
  - the number of comments left by users for the particular movie (e.g. "Comments(0)", "Comments(4)");
  - the list of comments left by users for the particular movie;
  - the input field where a user can submit a new comment;
- tests for all counters (items, likes, comments) pass and coverage is 100%.

**Additional features:**
- basic input validation is performed and error messages are thrown when the input is invalid;
- there are additional links in the navbar that list the movies depending on the genre (but not all of them as TVmaze API doesn't return more than 10 items per query);

Deployed with `gh-pages`:  https://arnoldnekemiah.github.io/Metflix/
Our project [presentation is here](https://drive.google.com/file/d/1j-jZmxtSH0VKtKJwpB-yiFXyGLYGiflC/view?usp=sharing).

### CODE REVIEW

File: src/item-counter.js
Comment (cynthiainga) [inline src/item-counter.js:1]: - [x] Good job so far but as mentioned in the project all counter functions need to be covered with unit tests using Jest, please kindly add the test for `itemCounter` 🙃 😁

File: src/modules/render-popup.js
Comment (cynthiainga) [inline src/modules/render-popup.js:1]: - [x] Good jog so far but there are still things to improve about your popup and you are good to go. When I open the pop if the item is not one of the three first I need to scroll up to see the popup window. Normally when opening the popup window it should open directly to the screen and we can see it no need to scroll the popup window should be fixed if we scroll down or up only the element behind the popup window can move not the popup. Kindly fix that 🙏🏾 
I take another project just to show you what I meant 🙃 

Your popup window

https://github.com/arnoldnekemiah/Metflix/assets/74501523/9079943c-6ccc-46d2-8325-707e5e9cf4af

How normally your popup should work

https://github.com/arnoldnekemiah/Metflix/assets/74501523/73b29323-d852-4632-b186-2024b88a2d12

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Zilola-Nazarova & @arnoldnekemiah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] There is a home page ✔️ 
- [ ] There is a comments popup. ✔️ 
- [ ] Project use ES6 and Webpack ✔️ 
- [ ] No linters errors ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/render-popup.js
Comment (Zilola-Nazarova) [reply src/modules/render-popup.js:1]: Thank you for this catch! I learned more about positioning while trying to fix that 😃

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢 🟢 🟢

Hi @Zilola-Nazarova & @arnoldnekemiah,

Great work done 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Implement comment and like features correctly 👍🏽
✅ Comment popup modal works fine and is nicely positioned 👍🏽
✅ Implement test for counters and all tests passed successfully 👍🏽
✅ Good commit messages and PR description 👍🏽

## Recommendation:
✅ For such great work, I would recommend you make your names in the footer be linked to your portfolio website individually so if you end up hosting and sharing this project, one would want to clink on your names to view your portfolio website to contact you should they need to 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**